community/get-in-touch.md

```diff
@@ -0,0 +1,101 @@
+---
+sidebar_position: 5
+---
+
+# Get in Touch
+
+The community at conda-forge would be happy to connect with you. We have
+got several ways for you to communicate with us, primarily through issue
+trackers, mailing lists, and real-time chat.
+
+## Issue Trackers
+
+The main issue trackers that you will interact with are
+
+- [staged-recipes](https://github.com/conda-forge/staged-recipes/issues):
+  You'll use staged-recipes to create a new conda package on conda-forge
+- [Our main docs
+  repo](https://github.com/conda-forge/conda-forge.github.io/issues):
+  You'll use this repo as a catch-all for issues where you're not sure
+  where else to put them
+- [Our enhancement proposals
+  repo](https://github.com/conda-forge/cfep/issues): You'll use the
+  enhancement proposals repo if you're interested in substantially
+  changing the way conda-forge operates.
+
+## Discussion Forum
+
+The primary discussion areas that you will engage with is
+
+- [discourse
+  forum](https://conda.discourse.group/c/pkg-building/conda-forge/25):
+  We are using discourse as our main communication channel. To ask a
+  question regarding usage of conda we encourage posting to our
+  Discourse forum.
+
+## Gitter and Element
+
+The main chat rooms that you'll interact with are
+
+- [gitter:
+  general](https://gitter.im/conda-forge/conda-forge.github.io): Our
+  general chat room for all things conda-forge. Pretty much any question
+  can be asked here and others in the community may be able to help.
+  Move stuff to an issue tracker if your question isn't getting
+  resolved.
+- [gitter: core/private](https://gitter.im/conda-forge/core): Private
+  chat for the @conda-forge/core team to discuss continued operations
+  and improvements to conda-forge.
+- [gitter: power pc](https://gitter.im/conda-forge-ppc64le/Lobby):
+  Public chat room for all things related to building for power pc (IBM)
+  systems.
+- [gitter: bot
+  subteam](https://gitter.im/conda-forge/regro-cf-autotick-bot): Public
+  chat room for all things related to the conda-forge automation
+  infrastructure. Our automation infrastructure is colloquially referred
+  to as "the bot".
+- [gitter:
+  compilers](https://gitter.im/conda-forge/conda-forge-compilers):
+  Public chat room focused on the conda-forge compiler stack.
+- [element:
+  conda-forge-space](https://app.element.io/#/room/#conda-forge-space:matrix.org):
+  An element space listing all conda-forge rooms.
+- [element:
+  interns](https://app.element.io/#/room/#conda-forge_conda-forge-interns:gitter.im):
+  Public chat room for conda-forge interns to discuss projects and seek
+  guidance from the community.
+- [element:
+  general](https://app.element.io/#/room/#conda-forge:matrix.org):
+  Public chat room for the conda-forge community to discuss anything
+  related to conda-forge.
```

Comment:
The line wrap is a bit too strict with these line breaks. Can we run `pandoc` with a more generous line length?